en/advanced-gimbal-control.md

```diff
@@ -1,20 +1,24 @@
 # Gimbal Control Setup
 
-PX4 contains a generic mount/gimbal control driver with different input and output methods. Any input can be selected to drive any output.
-
-First, make sure the driver runs, using `vmount start`, then configure its parameters.
+PX4 contains a generic mount/gimbal control driver with different input and
```

Comment:
What is the purpose of a gimbal control driver? Ie it is often worth saying something along the line of what you can control here - i.e. When you would need to do this setup and what config is reasonable - e.g. I assume you're doing this because you have a new gimbal and the sort of thing you want to define is what axis it can work with? Or is this more a tuning thing. Does that make sense?